packages/block-editor/src/components/block-draggable/dropping-insertion-point.native.js

```diff
@@ -95,9 +107,25 @@ export default function DroppingInsertionPoint( {
 			? findBlockLayoutByClientId( blocksLayouts.current, nextClientId )
 			: null;
 
-		const nextPosition = previousElement
-			? previousElement.y + previousElement.height
-			: nextElement?.y;
+		const elementsPositions = {
+			top: parseInt(
+				previousElement
+					? previousElement.y + previousElement.height
+					: nextElement?.y,
+				10
+			),
+			bottom: parseInt(
+				nextElement
+					? nextElement.y
+					: previousElement.y + previousElement.height,
+				10
+			),
```

Comment:
Not really, it was just to remove the decimals because sometimes you could get:

```
top 462.33331298828125
bottom 462.3333435058594
```

I changed it to `Math.floor` here https://github.com/WordPress/gutenberg/pull/40510/commits/008dfde0b92da42622dc657100b866cff3ff245b